Lib/test/test_float.py

```diff
@@ -949,6 +949,8 @@ def test_None_ndigits(self):
             self.assertEqual(x, 2)
             self.assertIsInstance(x, int)
 
+    def test_round_with_none_arg_direct_call(self):
+        self.assertEqual((1.0).__round__(None), 1.0)
```

Comment:
```suggestion
        for val in [(1.0).__round__(None),
                    round(1.0),
                    round(1.0, None)]:
        self.assertEqual(val, 1)
        self.assertIs(type(val), int)
```

This should be even better. Please, change the second test as well.